homeassistant/components/webhook/__init__.py

```diff
@@ -106,6 +108,26 @@ async def async_handle_webhook(
         _LOGGER.debug("%s", content)
         return Response(status=HTTPStatus.OK)
 
+    # POST & HEAD are considered simple requests. They do not trigger a CORS preflight
```

Comment:
Meh, I forgot about the simple requests. I don't feel like rolling our own CROS is a good way to go.

Do we really need it given the frontend change enforcing that we push for long random strings.

Reply:
I actually feel this PR isn't needed if everyone used long random strings from the start. But there are tutorials online that recommend short names, and those names are likely going to be the same for everyone that followed the tutorial. 

If you have a few moments, could you review the comments in #56446? That's how I got started down this path. GET is somewhat more dangerous because it's [trivial](https://developer.mozilla.org/en-US/docs/Web/Security/Referer_header:_privacy_and_security_concerns#how_can_we_fix_this) to avoid sending the referer header too.

My day job focuses on keeping users safe online. So it's easy for me to fall into the trap of preferring security over usability. If you could review the comments in #56446, and provide a bit of direction, that would be helpful for me. I'd rather not go down a path that makes things harder for users or hurts usability if you think it isn't needed.

Reply:
I suggest we close this PR and instead go for the method picker that you showed a screenshot off. Although I would even move that into the overflow menu so that we don't have to confuse the user always with it.

Reply:
Closing this. Thank you for taking a look and the feedback. I appreciate it!

I'll work on polishing-up the method/local options and try to have something ready early next week.